### PR TITLE
[TOD] Add zoo models for TOD pre-training; update README

### PR DESCRIPTION
**Pull request by moyapchen**
Having open source pre-trained models are nice.

## Test Plan 
Ran the pre-trained model loading commands locally to make sure they worked in terms of loading things from AWS 
```
parlai dd -t google_sgd_simulation_splits:OutDomainSystemTeacher -mf zoo:tod/tod_base_yes_api/model  --skip-generation false --api-schemas true
```
```
parlai dd -t google_sgd_simulation_splits:OutDomainSystemTeacher -mf zoo:tod/tod_base_no_api/model  --skip-generation false --api-schemas false
```

Otherwise docs are docs. 

(Note that there's a step in the original "model upload" internal ParlAI instructions that talks about changing permissions of files on S3 that I did not see in the new AWS SSO instructions; not quite sure what to do about that.)

### CODE REVIEW

File: parlai/zoo/tod/tod_base_yes_api.py [hunk 1]
```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env python3
+
+# Copyright (c) Facebook, Inc. and its affiliates.
+# This source code is licensed under the MIT license found in the
+# LICENSE file in the root directory of this source tree.
+"""
+Task-Oriented Dialog pretrained model *without* usage of API Schemas.
```
Comment (stephenroller) [inline parlai/zoo/tod/tod_base_yes_api.py:7]: we should probably be more specific. your paper will not be the only tod paper.

Comment (moyapchen) [reply parlai/zoo/tod/tod_base_yes_api.py:7]: Good catch

File: projects/tod_simulator/README.md [hunk 2]
```diff
@@ -71,9 +87,63 @@ Given a conversation generated from `tod_world_script`, outputs statistics about
...
+Multi-step train, evaluation, and data generation script used in Simulations paper. Uses Google SGD Out-of-Domain as primary dataset; note "STANDALONE\_API\_FILE\_PATH" that needs to be set in file. Makes use of `do_get_passing_only_on_dir.py` and `get_al_samples_for_gsgd.py`; use `get_passing_only.py` and `get_api_data.py` after the fact for analysis. 
 
 Note that this script is intended to be run in a SLURM environment matching that of the Simulations paper authors. It is unknown how the script performs in other settings but is included as a reference.
+
+## Tasks used in the paper 
+
+See the appendix of [the paper](https://arxiv.org/abs/2110.06905) (or the description of the task in ParlAI Task List) for explanations of these datasets. Below, we include the dataset name, the command to run the `SystemTeacher` relevant for each of the datasets, and any other notable details. Other agents and teachers for the dataset are specified in the relevant task `agent.py` files.
```
Comment (EricMichaelSmith) [inline projects/tod_simulator/README.md:99]: hmm maybe it's worth saying what SystemTeacher is? This terminology is used throughout the README

File: projects/tod_simulator/README.md [hunk 3]
```diff
@@ -71,9 +87,63 @@ Given a conversation generated from `tod_world_script`, outputs statistics about
...
+
+### Experimentation Tasks
+
+* Google SGD Out-of-Domain
+   * `parlai dd -t google_sgd_simulation_splits:OutDomainSystemTeacher`
+* MultiWoz (not currently included in paper) 
+   * `parlai dd -t multiwoz_v22:SystemTeacher`
+   * This is a preprocessing of the dataset based on MultiWoz v2.2. Though utterances are the same as used for pre-training, API Call and API Response structures aer different. 
+
+See "scripts in project directory" for scripts associated with training, evaluation, and data generation.
```
Comment (EricMichaelSmith) [inline projects/tod_simulator/README.md:129]: Super minor - maybe clarify that this refers to another section in this README, right?

**Review by EricMichaelSmith — COMMENTED** [review]
Minor quibbles, otherwise seems reasonable!

**Pull request closed without merging**